ion_hash_tests.ion

```diff
@@ -1013,61 +949,6 @@ structCount14::{
                    0x0e)
   ) }
 }
-// the following test fails if digests of struct fields are escaped prior to sorting;
-// struct field digests must be sorted first, then escaped
```

Comment:
Does this commented out test imply that the current implementation does not sort struct field correctly? Or is it that the test uses a feature we are not covering? If the test is using a feature we are not covering perhaps we can make a new test that still exercises the sorting requirement but does so with features we currently support?

Reply:
This is failing due to symbol support (ms). I will try and make a new test that exercises this sorting requirement

Reply:
Added a modified version of this scenario by replacing the symbol: ms to int: 0